account_chart_update/wizard/wizard_chart_update.py

```diff
@@ -375,7 +360,13 @@ def _get_real_xml_name(self, template):
         (name, module) = external_id.split(".")
         return "%s.%d_%s" % (name, self.company_id.id, module)
 
-    @api.multi
+    @tools.ormcache("templates")
+    def find_taxes_by_templates(self, templates):
```

Comment:
Is this method meant to be called from RPC?

- Yes: then decorate with the corresponding `@returns('self')`
- No: then prefix it with underline to make it private: `_find_taxes_by_templates`

Reply:
I think in v13 `@api.returns` is deprecated, and not sure why do you want to put `self` as argument. Any way, you did these methods (for v9 or for v10), so you tell us.

Reply:
It was 4 years ago, I have no idea :grin:

Reply:
OK, then discarding the suggestions. If we ever need that, we add it.